paper/paper.md

```diff
@@ -1,158 +1,74 @@
+To facilitate multi-centric research on medical imaging data the Kaapana framework (\[@usesMethodIn:scherer2020joint\]) was created. It provides an open-source framework based on cloud native technologies enabling the large-scale analysis of medical imaging data as well as sharing analysis methods across installations. Primary data as well as analysis results can be stored in the platform and metadata for example from DICOM images is extracted to support cross-center definition and curation of imaging patient cohorts.  Multi center data discovery, like in the DART project of Cancer Core Europe (CCE) also requires approaches for managing and sharing catalogs of metadata. .
 
-Automated processing workflows and Machine Learning make it instructable to combine different datasets from across the different research institutions and consortia. In the ideal world, it would then be possible to make automatic queries across the various research data repositories to pull the required images and the adjacent metadata automatically in one place with minimal manual involvence / correction. Unfortunately, that is not the case because of the variety of reasons.
+# **Goals**
 
-## Problems with FAIRness of bioimaging datasets
+During the hackathon, we assessed the implementation of the FAIR principles in the current Bioimaging and Clinical Imaging data repositories. Additionally, to make the RDF export triples from the IDR discoverable, we also explored the  Fair Data Point interface  (\[@citesAsAuthority:da2023fair\]), as an idea to facilitate the exposure of machine-actionable metadata and how it could be added to the Imaging Data Resource (IDR) portal. 
 
-### Tiers of the problems
+# **Results**
 
-In SWAT4HCLS this year, there were a lot of posters about FAIR Data Points (FDP). However, in the reality of "scraping web" for imaging data the FDPs are simply not provided in most of the cases. Some imaging datasets in life sciences (e.g. BABRI `[@citesAsDataSource:yang2021early]` ) have not even a clear way to access the data **manually**, i.e. the data is accessible on paper but in reality the website has been taken down, the corresponding author does not respond or similar.
+To classify the real-world problems with imaging datasets, there was an idea to work on a new “tier” system for publicly available datasets. This tier system would grade the datasets on a scale of their compliance with best data practices. It would complement  existing assessments like the FAIR principles or  the 5-star Open Data system (\[@discusses:Berners-Lee\_2009\]). While the existing tools assess the principal readiness of the data to be FAIR and interlinked, mainly from a technical perspective, the proposed tier system would focus on tangible amount of manual labor that is required to obtain necessary metadata and download public imaging datasets, e.g. finding the right contact person, filling out forms, machine access to metadata or the actual data etc. 
```

Comment:
Worth cross-linking on of your posters here?

Reply:
Great idea, since they are also on Zenodo 👍